combat-trainer.lic

```diff
@@ -1434,11 +1434,14 @@ class ManipulateProcess
     @threshold = settings.manipulate_threshold
     echo("  @threshold: #{@threshold}") if $debug_mode_ct
 
+    @construct_mode = settings.construct(false)
```

Comment:
What would you think about moving this into a predicate in `GameState`? There's a lot of duplication with `AttackProcess` here

Reply:
Sure thing.